addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
+            <channel id="ssnextsecond" typeId="superscheduleinfo">
+                <label>Super Schedule Next Seconds</label>
+                <description>Seconds until the next step in the super schedule</description>
+            </channel>
+        </channels>
+
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <context>network-address</context>
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the PRG access point</description>
+            </parameter>
+            <parameter name="userName" type="text">
+                <label>User Name</label>
+                <description>The user name to log in to the PRG access point (typically "nwk" for connection 1 or "nwk2" for connection 2)</description>
+                <default>nwk</default>
+            </parameter>
+            <parameter name="retryPolling" type="integer">
+                <label>Polling Interval to try to reconnect</label>
+                <description>Interval (in seconds) to try to (re)connect to the matrix</description>
+                <default>30</default>
+            </parameter>            
+            <parameter name="shadeZones" type="text">
+                <label>Shade Zones</label>
+                <description>Comma separated list of zones that are shades</description>
+                <default></default>
+            </parameter>            
+        </config-description>
+    </bridge-type>
+
+    <thing-type id="grafikeye">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="prgbridge"/>
+        </supported-bridge-type-refs>
+
+        <label>Grafik Eye</label>
+        <description>Controls a Grafik Eye</description>
+
+        <channels>
+            <channel id="scene" typeId="scene"/>
+            <channel id="scenelock" typeId="scenelock"/>
+            <channel id="sceneseq" typeId="sceneseq"/>
+            <channel id="zonelock" typeId="zonelock"/>
+            <channel id="zonelower1" typeId="zonerampdown"/>
+            <channel id="zonelower2" typeId="zonerampdown"/>
+            <channel id="zonelower3" typeId="zonerampdown"/>
+            <channel id="zonelower4" typeId="zonerampdown"/>
+            <channel id="zonelower5" typeId="zonerampdown"/>
+            <channel id="zonelower6" typeId="zonerampdown"/>
+            <channel id="zonelower7" typeId="zonerampdown"/>
+            <channel id="zonelower8" typeId="zonerampdown"/>
+            <channel id="zoneraise1" typeId="zonerampup"/>
+            <channel id="zoneraise2" typeId="zonerampup"/>
+            <channel id="zoneraise3" typeId="zonerampup"/>
+            <channel id="zoneraise4" typeId="zonerampup"/>
+            <channel id="zoneraise5" typeId="zonerampup"/>
+            <channel id="zoneraise6" typeId="zonerampup"/>
+            <channel id="zoneraise7" typeId="zonerampup"/>
+            <channel id="zoneraise8" typeId="zonerampup"/>
```

Comment:
If you could get them to write the API to stop a specific zone - I'd support it!  :)  Unfortunately there is no API for that - you'd have to use the ALL zone stop on the PRG unit itself to achieve that.  Want me to add a comment in the README.MD about that?